CollapseLauncher/XAMLs/MainApp/Pages/SettingsPage.xaml.cs

```diff
@@ -78,9 +78,12 @@ public SettingsPage()
             if (IsAppLangNeedRestart)
                 AppLangSelectionWarning.Visibility = Visibility.Visible;
 
-            if (IsChangeRegionWarningNeedRestart)
+            if (IsChangeRegionWarningNeedRestart) 
```

Comment:
unexpected space here

Reply:
Fixed in https://github.com/CollapseLauncher/Collapse/pull/449/commits/dd2901c974e260044f78024631e13d69988b9e12